### PR TITLE
feat: [STMS-2231] Github actions: `add-attachment`

### PR DESCRIPTION
**Pull request by lysoff**

### CODE REVIEW

**Comment by doniyor2109** [comment]
@lysoff Can we call it like this? `jira/add-attachment`

File: jira/add-build-attachment/action.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    description: 'JIRA issue'
+    required: true
+
+  path:
+    description: 'Build directory'
+    required: true
+
+  filename:
+    description: 'File name'
+    default: 'build.zip'
```
Comment (doniyor2109) [inline jira/add-build-attachment/action.yml:15]: Lets remove default as users should decide it

File: jira/add-build-attachment/index.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  if (previousAttachment) {
+    await jira.deleteAttachment(previousAttachment.id);
+  }
+
+  const destinationStream = fs.createWriteStream(filename);
+
+  const archive = archiver('zip');
+  archive.on('error', (err) => {
+    console.error('Error while zipping', err);
+  });
```
Comment (doniyor2109) [inline jira/add-build-attachment/index.ts:29]: Do you think we should archive attachments? Can we upload them directly?

Comment (lysoff) [reply jira/add-build-attachment/index.ts:29]: @doniyor2109 not necessarily, but in case of builds we probably should

**Pull request merged**